sopel/cli/config.py

```diff
@@ -75,15 +77,16 @@ def handle_list(options):
     It is possible to filter by extension using the ``-e/--ext/--extension``
```

Comment:
Hmm, spotted another of this pattern I just mentioned in my review of #1646. Not necessary to change it in this PR, but maybe you want to do so in the other. 😸